outputs.tf

```diff
@@ -1,14 +1,18 @@
-output "id" {
-  description = "ID of the created example"
-  value       = module.this.enabled ? module.this.id : null
-}
-
-output "example" {
-  description = "Example output"
-  value       = module.this.enabled ? local.example : null
-}
+output "region_az_alt_code_maps" {
+  description = <<-EOT
+    Collection of maps converting between official AWS Region, Availabiltiy Zone, and Local Zone codes and shorter unofffical codes without hyphens. Inspired for use in naming and tagging so that region or AZ code will be 1 semantic unit.
```

Comment:
```suggestion
    Collection of maps converting between official AWS Region, Availability Zone, and Local Zone codes and shorter unofffical codes without hyphens. Inspired for use in naming and tagging so that region or AZ code will be 1 semantic unit.
```